### DISCUSSION TITLE
Wording in the script comments: hibernate

### DISCUSSION THREAD

**Discussion by juanriccio** [farag2/Sophia-Script-for-Windows#219] [Q&A] [answered]
```
# Disable hibernation. Do not recommend turning it off on laptops
# Отключить режим гибернации. Не рекомендуется выключать на ноутбуках
Hibernate -Disable

```
It is not clear from the wording: should I KEEP this line (disable hibernate), or COMMENT this line (enable hibernate) on my laptop?
In other words, I don't understand what I shouldn't turn off: the script line, or the hibernate feature. Maybe the comment could be reworded for extra clarity.

And, while we're at it, what is the problem with hibernate/not-hibernate on laptops?

Sorry if the question is stupid :-)

**Comment by farag2** [comment, marked as answer]
No, your question isn't stupid. :) For me I don't understand why you didn't catch the point. There is a feature to turn off hibernation. I don't recommend to turn it off on laptops, because every time you close a lid, it has to be hibenated or go to sleep depending on what you prefer. It frees some spaces, but the cost will be that your device won't be able to hibernate. :)
If the feature isn't commented out by default, I recommend to turn it off.

**Comment by juanriccio** [comment]
OK I got it, thanks @farag2. I suggest rewording that stanza as follows (using the same verb "disable" after "not recommend");

```
# Disable hibernation. Do not recommend disabling it on laptops
# Отключить режим гибернации. Не рекомендуется отключить на ноутбуках
Hibernate -Disable
```

**Reply by farag2** [reply]
Going to release 5.12 in an hour. :)
Plus the 6.0 version for Windows 11 as a separate archive.

**Reply by juanriccio** [reply]
Wow, that's some quick/frequent update! I'm actually using v5.2.10 (on LTSC here, so I can't use 5.11/5.12). Thanks again for this tool. It cut my time to setup a new machine in half! :-)
